### DISCUSSION TITLE
Githu. Student developer pack

### DISCUSSION THREAD

**Discussion by yunusemremaral** [community/community#84121] [GitHub Education]
How can I prove that I am a student? Photos are not accepted. Can you help me guys.

**Comment by fury-05** [comment]
You can use you academic email from college / university you took admission, if you don't have ask them for one. Remember your university must be having tie up with GitHub education to avail this benefits.

After adding or registering by academic email you can upload you ID card or Study Certificate which has you name and other details visible and should match with the details you have given.

*Note: You need to be currently enrolled in university with student status.

Hope this solves your query.

**Reply by yunusemremaral** [reply]
Thanks bro

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @ImGozluklu , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
